posthog/models/user.py

```diff
@@ -136,6 +136,7 @@ class User(AbstractUser, UUIDClassicModel):
     toolbar_mode: models.CharField = models.CharField(
         max_length=200, null=True, blank=True, choices=TOOLBAR_CHOICES, default=TOOLBAR
     )
+    is_email_verified: models.BooleanField = models.BooleanField(default=False, null=True, blank=True)
```

Comment:
I'm getting an error with validating Django migrations saying:

`Found a non-null field or default added to an existing model. This will lock up the table while migrating. Please add 'null=True, blank=True' to the field`

But I've done that here... Is it because I have `default=False`?